index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+  <body>
+      <header>
+        <div class="toolbar">
+            <div>
+                <h1>Ishmael</h1>
+            </div>
+            <div>
+                <img src="img/union.png" alt="">
+            </div>
+        </div>
+        <div class="headline">
+            <div>
+                <h2>Hey There. I’m Alex</h2>
+                <h3>I’am a Software Developer</h3>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+                see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+                <ul>
+                    <li><img src="img/GitHub.png" alt=""></li>
+                    <li><img src="img/Linkedin.png" alt=""></li>
+                    <li><img src="img/Angelist.png" alt=""></li>
+                    <li><img src="img/Twitter.png" alt=""></li>
+                    <li><img src="img/Medium.png" alt=""></li>
+                </ul>
+            </div>
+        </div>
+      </header>
```

Comment:
- [ ] Good looking toolbar 👍 however the `<header>` tag should only be used for the toolbar part of the code and inside there should be a `<nav>` tag, something like in the below example
```
<header>
  <nav>
    ...
  </nav>
</header>
```